examples/gatsby-sourcing-wordpress/src/templates/post.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react"
+import { graphql } from "gatsby"
+import PropTypes from "prop-types"
+
+class Post extends Component {
+  render() {
+    const post = this.props.data.wordpressPost
+
+    return (
+      <Layout>
+        <h1>{post.title }<h1/>
+        <div>{post.content}<div/>
+    )
```

Comment:
i miss here a closing `</Layout>`

Reply:
Must've missed these while reviewing. Thanks for pointing these out.